src/sentry/event_manager.py

```diff
@@ -2044,49 +2045,46 @@ def _process_existing_aggregate(
 
 def _get_severity_score(event: Event) -> float | None:
     op = "event_manager._get_severity_score"
-    logger_data = {"event_id": event.data["event_id"], "op": op}
+    title = ErrorEvent().compute_title(dict(event.get_event_metadata()))
```

Comment:
This is a bit strange, no? Instantiating a class just to call a function? I'm also not finding any other uses of this `compute_title()` function which makes me wonder if it's the right thing to use.

Can you use `event.title`? Looking at the base class it looks like that might be the way to use this function.

Reply:
Yeah, the method should really be static, since `ErrorEvent` doesn't actually have any data of its own. In the next PR I'm adding a helper method to look for a stacktrace, and that will be static. 

For this, though, you're right, using `event.title` is simpler (assuming this happens after `event.title` is computed). Because I was computing the payload data manually before this change (in order to avoid truncation), I think I just defaulted to continuing to do so. I'll check if `title` is available at this point in the pipeline and switch to using that if it is.